script/server/cli.ts

```diff
@@ -2,8 +2,15 @@
 import { argv } from 'yargs';
 import server from '.';
 
-server({port: parseInt(process.env.PORT || '', 10), ...argv})
-  .then(({port}) => console.log(`running on port: ${port}`))
+const port = parseInt(process.env.PORT || '', 10);
+
+if (!process.env.PORT || Number.isNaN(port)) {
```

Comment:
you shouldn't need to make these changes, it should get caught by https://github.com/openstax/rex-web/blob/master/script/server/index.ts#L22

i would prefer not to add the check here because when i run it manually i like to specify the port with `--port` rather than the env